ethereum/referencetests/build.gradle

```diff
@@ -66,13 +97,15 @@ sourceSets {
       runtimeClasspath += main.output
       srcDirs "src/reference-test/java",
         blockchainReferenceTests,
+        execSpecTests,
         generalstateReferenceTests,
         generalstateRegressionReferenceTests
     }
     resources {
       srcDirs 'src/reference-test/resources',
         'src/reference-test/external-resources',
-        'src/reference-test/templates'
+        'src/reference-test/templates',
+        'build/tmp/execution-spec-tests/'
```

Comment:
why do we need this extra directory?

Reply:
I see the files exist twice locally...
```
find . -name withdrawals_newly_created_contract.json
./ethereum/referencetests/build/resources/referenceTest/fixtures/withdrawals/withdrawals/withdrawals_newly_created_contract.json
./ethereum/referencetests/build/tmp/execution-spec-tests/fixtures/withdrawals/withdrawals/withdrawals_newly_created_contract.json
```

Reply:
I suspect that that the resources folder gets 'copied' from src at a later stage ... because if I don't include the build/tmp path, the relavant json files doesn't get visible.